core/util/extendConfig.js

```diff
@@ -59,17 +62,25 @@ function htmlReport(config, userConfig) {
   config.compareReportURL = path.join(config.html_report, 'index.html');
 }
 
-function comparePaths(config) {
+function comparePaths (config) {
   config.comparePath = path.join(config.backstop, 'compare');
   config.tempCompareConfigFileName = temp.path({suffix: '.json'});
 }
 
-function captureConfigPaths(config) {
-  config.captureConfigFileName = path.join(config.backstop, 'capture', 'config.json');
+function captureConfigPaths (config) {
+  var captureDir = path.join(tmpdir, 'capture');
+  if (!fs.existsSync(captureDir)) {
+    fs.mkdirSync(captureDir);
+  }
+  var tempCompareConfigFileName = config.tempCompareConfigFileName;
+  config.tempCompareConfigFileName = '';
+  var configHash = hash(config);
+  config.tempCompareConfigFileName = tempCompareConfigFileName;
```

Comment:
It's now my own opinion speaking (you may see it in a different way):
but why do we need the temporary config after the test? Cause we allready got the config file. It's not a great deal to generate a temporary file again. 
It's just a waste of space, if we're about to run multiple tests as we got many different settings, we get a huge load of files. So why we won't keep it really temporary, as the filename intends and delete after the execution? With the hashed value the timestamp and the config, we guarantee that we only use the temporary file as long as the test will be executed and it won't remains dead on our temp directory. 
An automatically garbage collection of our temporary testfiles would be nice.